dex/testing/dcr/create-wallet.sh

```diff
@@ -60,8 +60,8 @@ if [ "${ENABLE_VOTING}" = "2" ]; then
 cat >> "${WALLET_DIR}/${NAME}.conf" <<EOF
 enablevoting=1
 enableticketbuyer=1
-ticketbuyer.limit=6
-ticketbuyer.balancetomaintainabsolute=1000
+ticketbuyer.limit=7
```

Comment:
Did you ultimately need to bump this?  I expect the real problem was the combination of the `ticketbuyer.balancetomaintainabsolute` line as well as sending all the coins to `beta` too early.

Reply:
6 was still fine. The real problem was a misplaced sendtoaddress that was spending 1000 DCR after block 100, but this `ticketbuyer.balancetomaintainabsolute` didn't help either.